module/VuFind/src/VuFind/RecordDriver/EDS.php

```diff
@@ -567,7 +607,235 @@ protected function getSupportedCitationFormats()
      */
     public function exportDisabled($format)
     {
-        // EDS is not export-friendly; disable all formats.
-        return true;
+        // at the moment we only support RIS
+        return strtolower($format) != 'ris';
+    }
+    
+     /**
+     * This method can fetch fields out of the deeply nested RecordInfo array. 
+     * many fields are here in a more readable and atomic format
+     * 
+     * The latter two params are only for internal use (recursion)
+     *  
+     * @param array $arrayKeys Key path to the needed value
+     * @param int   $level     only used for recursion
+     * @param array $fields    only used for recursion
+      * 
+     * @return array|string
+     */
+    protected function getFieldRecursive($arrayKeys, $level = 0, $fields = null)
+    {
+        // special case: we have to iterate through items array
```

Comment:
I'd prefer the colon separator as the slash looks like a PHP class. 
I'm really sorry, I can't help here. My boss decided to get me into another team so I'm mostly  out of the VuFind stuff :-(